### PR TITLE
Drop embedded Provision

### PR DESCRIPTION
**Pull request by james-nesbitt**
- drop provision module (using local path until it is published)

ALSO

- fix some of the k0sctl firewall setting
- fix the chart scripts to allow the custom my_tf.cfg file to be detected and included.

### CODE REVIEW

File: launchpad-stack-aws3/infra/provision.tf [hunk 1]
```diff
@@ -6,7 +6,7 @@ locals {
 
 # PROVISION MACHINES/NETWORK
 module "provision" {
-  source = "../../provision"
+  source = "/home/jnesbitt/Documents/Mirantis/tools/terraform-mirantis-provision-aws"
```
Comment (pgedara) [inline launchpad-stack-aws3/infra/provision.tf:9]: Incorrect module path

Comment (james-nesbitt) [reply launchpad-stack-aws3/infra/provision.tf:9]: whoops - 1 min

**Review by pgedara — CHANGES_REQUESTED** [review]
Module path for platform module in launchpad-stack-aws3/infra/platform.tf needs to be corrected.

File: launchpad-stack-aws3/infra/provision.tf (hunk 1 above)
Comment (james-nesbitt) [reply launchpad-stack-aws3/infra/provision.tf]: fixed

**Review by pgedara — APPROVED** [review]

**Pull request merged**